src/models/group/controller.spec.ts

```diff
@@ -0,0 +1,115 @@
+
+import controller from './controller';
+import service from './service';
+// eslint-disable-next-line no-unused-vars
+import { IGroup } from './IGroup';
+
+describe('Group Controller', () => {
+    const mock = jest.fn();
+    const testError = new Error('test');
+    const errorMock = jest.fn(() => {
+        throw testError;
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+    describe('add', () => {
```

Comment:
I would rename test cases in a real app of course :)

Reply:
Inner "it" descriptions are showing what test is about.
So for first test it would be "Group Controller add should correctly process valid input"